### PR TITLE
use golang.org/x/sys on ppc64, ppc64le

### LINKED ISSUE #19: Use x/sys instead of std syscall
I meet the problem when I use this package on ppc64x. After debugging, I find it's the std syscall's fault. And I raised it on https://github.com/golang/go/issues/22000
The syscall.TCGETS on ppc64x is broken.

However in https://github.com/golang/go/issues/19560 this problem has been raised, but still not fixed in lasted Go release. And in the comments on https://github.com/golang/go/issues/19560, the core developers seems suggesting people to use x/sys instead. So people don't need to wait a new release of Go to get this fixed.

### PR DESCRIPTION
**Pull request by mattn**
@zhsj could you please try this PR?
close #19

### CODE REVIEW

File: isatty_linux_ppc64.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// +build linux,ppc64
```
Comment (zhsj) [inline isatty_linux_ppc64.go]: ppc64le can't see this file, the filename should not be _ppc64, maybe _ppc64x.go is fine. And the tag should add linux,ppc64le too.

File: isatty_others.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // +build !windows
-// +build !appengine
+// +build !appengine,!ppc64,!ppc64le
```
Comment (zhsj) [inline isatty_others.go]: why ppc64x is excluded here?
otherwise ppc64x miss IsCygwinTerminal

File: isatty_linux_ppc64.go (hunk 1 above)
Comment (mattn) [reply isatty_linux_ppc64.go]: Sorry, I can't reproduce it because I don't have ppc64 or ppc64le. Do this occur only on ppc64le?

Comment (zhsj) [reply isatty_linux_ppc64.go]: Hi, ppc64 and ppc64le both have the problem.

Comment (mattn) [reply isatty_linux_ppc64.go]: Oops. will fix.

**Comment by zhsj** [comment]
hi, maybe not clear, when I say ppc64x, it's ppc64 + ppc64le.. so the last commit seems fail to ppc64...

File: isatty_others.go [hunk 3]
```diff
@@ -1,5 +1,7 @@
 // +build !windows
 // +build !appengine
+// +build !ppc64
+// +build !ppc64le
```
Comment (zhsj) [inline isatty_others.go]: Hi, still don't know why you want to exclude `IsCygwinTerminal` function on ppc64 and ppc64le. I think this file doesn't need change.

Comment (mattn) [reply isatty_others.go]: I was confused.

**Comment by zhsj** [comment]
thanks for the effort :) looks good to me now. And I have tested it on ppc64 and ppc64le, both work fine

**Comment by mattn** [comment]
Thank you

**Pull request merged**